catalyst-gateway/bin/src/db/event/config/jsonschema/frontend.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$id": "https://www.stephenlewis.me/sentry-schema.json",
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Frontend JSON schema",
+	"type": "object",
+	"properties": {
+		"sentry_dsn": {
+			"$ref": "#/definitions/httpsUrl",
+			"description": "The Data Source Name (DSN) for Sentry."
+		},
+		"sentry_release": {
+			"type": "string",
+            "description": "A version of the code deployed to an environment"
```

Comment:
Lets make the schema more like this:
```
{
  "sentry" : {
    "dsn" : <url>,
    "release" : <string>,
    "environment": <string>,
  }
}
```
We don't explicitely need `"additionalProperties":true` because its the default, as is the fields being optional.
So to be consistent, lets only use `additionalProperties` if we want to restrict additional properties from being added, which normally we would not want to do.
Also, fix up the indentation.